kasa/smartbulb.py

```diff
@@ -142,8 +142,11 @@ async def get_light_state(self) -> Dict[str, Dict]:
         # TODO: add warning and refer to use light.state?
         return await self._query_helper(self.LIGHT_SERVICE, "get_light_state")
 
-    async def set_light_state(self, state: Dict) -> Dict:
+    async def set_light_state(self, state: Dict, *, transition: int = None) -> Dict:
```

Comment:
Not really part of this PR, but shouldn't we further specify what is in the output `Dict`?

Reply:
That could be added, it'd be `Dict[str, Union[str,Dict]]` or `Dict[str, Union[str,Dict[str,Union[str,int]]]` (which is not very beautiful :-)

Here's an example what it returns:
```
{'on_off': 0, 'dft_on_state': {'mode': 'normal', 'hue': 0, 'saturation': 0, 'color_temp': 2000, 'brightness': 23}}

```

Reply:
Right, that wouldn't be easy to read 😅